src/uikit.cs

```diff
@@ -6923,12 +6933,48 @@ interface UIActivityIndicatorView : NSCoding {
 	}
 #endif // !WATCH
 
+	[NoWatch, NoTV]
+	[iOS (11,0)]
+	[Protocol]
+	interface UIItemProviderPresentationSizeProviding {
+		[Abstract]
+		[Export ("preferredPresentationSizeForItemProvider")]
+		CGSize PreferredPresentationSizeForItemProvider { get; }
+	}
+
+	[iOS (11,0), TV (11,0), Watch (4,0)]
+	[Category]
+	[BaseType (typeof(NSItemProvider))]
+	interface NSItemProvider_UIKitAdditions {
+		[NoWatch, NoTV]
+		[NullAllowed, Export ("teamData", ArgumentSemantic.Copy)]
+		NSData GetTeamData ();
```

Comment:
Maybe this should be inlined in `NSItemProvider` instead, that would make it possible to use actual properties instead of Get/Set methods. @spouliot, what do you think?